include/isaac/value_scalar.h

```diff
@@ -34,7 +34,7 @@ namespace isaac
 class scalar;
 class expression_tree;
 
-union ISAACAPI values_holder
+struct ISAACAPI values_holder
 {
```

Comment:
Why couldn't we keep a union here? Using a struct instead will multiply the size of the class by 4, which will increase the (already high) overhead associated with expression trees construction and parsing